packages/runtime/container-runtime/src/gc/gcTelemetry.ts

```diff
@@ -69,7 +69,7 @@ interface INodeUsageProps extends ICommonProps {
  * Encapsulates the logic that tracks the various telemetry logged by the Garbage Collector. There are 4 types of
  * telemetry logged:
  * 1. inactiveObject telemetry - When an inactive node is used - A node that has been unreferenced for inactiveTimeoutMs.
- * 2. sweepReadyObject telemetry - When a sweep ready node is used - A node that has been unreferenced for sweepTimeoutMs.
+ * 2. sweepReadyObject telemetry - When a sweep ready node is used - A node that has been unreferenced for tombstoneTimeoutMs.
```

Comment:
This comment needs to be updated to talk about both tombstoneReady and sweepReady telemetry because we log both now.